code/game/objects/items/cards_ids.dm

```diff
@@ -376,6 +376,11 @@
 	// Hard reset access
 	access.Cut()
 
+/// Clears the economy account from the ID card.
+/obj/item/card/id/proc/clear_account()
```

Comment:
Setters like this are generally weird, but I'll let it slide since it's handling internals in a stable manner.